sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md

```diff
@@ -1,34 +1,23 @@
-## Microsoft Azure SDK for Python
+# Microsoft Azure SDK for Python
 
 This is the Microsoft Azure Personalizer Client Library.
+This package has been tested with Python 2.7, 3.5, 3.6, 3.7 and 3.8.
+For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
 
-This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
 
-For a more complete set of Azure libraries, see the
-[azure sdk python release](https://aka.ms/azsdk/python/all).
+# Usage
 
-## What is Personalizer?
 
-  - [What is
-    Personalizer?](https://docs.microsoft.com/azure/cognitive-services/personalizer/what-is-personalizer)
-  - [How to create a Personalizer
-    service](https://docs.microsoft.com/azure/cognitive-services/personalizer/how-to-settings)
 
-## Usage
 
-[Quickstart: Personalize client library for
-Python](https://docs.microsoft.com/azure/cognitive-services/personalizer/sdk-learning-loop?pivots=programming-language-python)
+For code examples, see [Personalizer](https://docs.microsoft.com/python/api/overview/azure/) on docs.microsoft.com.
 
-## Documentation
 
-For SDK reference documentation, see
-[Personalizer](https://docs.microsoft.com/python/api/overview/azure/) on
-docs.microsoft.com.
```

Comment:
@weshaggard this commit was automated by the CI and it removes material that, presumably, should remain. Is there a way to disable this?

Reply:
I'm not sure what automation generated this readme. @lmazuel is this being generated by autorest?

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/18720/commits/b1c50c4ae51f9d4c152d333cb75bd4aab5a4982a
This entire commit was made by automation, but I'm not sure which (or why for that matter).